syndeo-template/src/main/java/com/google/cloud/syndeo/transforms/pubsub/SyndeoPubsubReadSchemaTransformProvider.java

```diff
@@ -178,12 +184,27 @@ public PCollectionRowTuple expand(PCollectionRowTuple input) {
   @DefaultSchema(AutoValueSchema.class)
   @AutoValue
   public abstract static class SyndeoPubsubReadSchemaTransformConfiguration {
+    @SchemaFieldDescription(
+        "The name of the topic to consume data from. If a topic is specified, Syndeo"
+            + " will create a new subscription for that topic and start consuming from that point. "
+            + "Format: projects/${PROJECT}/topics/${TOPIC}")
     public abstract @Nullable String getTopic();
 
+    @SchemaFieldDescription(
+        "The name of the subscription to consume data. "
```

Comment:
Do we also want to mention that either a topic or a subscription should be provided? 

"The name of the subscription to consume data from. Either a topic or subscription must be provided. "

Reply:
done!